js/src/views/Account/account.js

```diff
@@ -74,10 +74,15 @@ class Account extends Component {
   componentWillReceiveProps (nextProps) {
     const prevAddress = this.props.params.address;
     const nextAddress = nextProps.params.address;
+    const { accounts } = nextProps;
 
     if (prevAddress !== nextAddress) {
       this.setVisibleAccounts(nextProps);
     }
+
+    if (!Object.keys(this.exportStore._accounts).length) {
```

Comment:
Might be good to whether have a getter, or rename the `_accounts` variable `accounts`. We try to use `_` for variables to shouldn't be accessed from outside of the classe